student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(classroom, age, name = 'unkwomn', parent_permission: true)
+    super(name, age, parent_permission)
```

Comment:
### Optional
- [ ] It would be better if our initialize method and our super method follows the same order. For example;
```
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
end
```